src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -44,6 +44,33 @@
         EMAIL_SEND_DATE,
         EMAIL_CONTENT
     }
+    public static final String SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE =
+            "<b>%s</b> cannot be empty and must be no longer than %d characters.";
+    public static final String SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE_WITHOUT_HTML =
+            "%s cannot be empty and must be no longer than %d characters.";
+
+    public static final String SIZE_CAPPED_POSSIBLY_EMPTY_STRING_ERROR_MESSAGE =
+            "<b>%s</b> must be no longer than %d characters.";
+
+    public static final String ALPHANUMERIC_STRING_ERROR_MESSAGE =
+            "<b>%s</b> can only contain alphabets, numbers and whitespaces.";
+
+    public static final String INVALID_NAME_ERROR_MESSAGE =
+            "<b>%s</b> must start with an alphanumeric character, "
+            + "and cannot contain any vertical bar (|) or percent sign (%%).";
+    public static final String INVALID_NAME_ERROR_MESSAGE_WITHOUT_HTML =
+            "%s must start with an alphanumeric character, "
+            + "and cannot contain any vertical bar (|) or percent sign (%%).";
+
+    public static final String WHITESPACE_ONLY_OR_EXTRA_WHITESPACE_ERROR_MESSAGE =
+            "<b>%s</b> must not contain only whitespace or extra spaces at the beginning or at the end of the text.";
+
+    public static final String NON_HTML_FIELD_ERROR_MESSAGE =
+            "<b>%s</b> must not contain the following special html characters in brackets: "
+            + "(&lt; &gt; \\ &#x2f; &#39; &amp;)";
+
+    public static final String NON_NULL_FIELD_ERROR_MESSAGE =
+            "<b>%s</b> must not be empty.";
```

Comment:
I shifted the more generic error messages up here from below, so the subsequent error messages can be built off of these.